tests/model_registry/rest_api/conftest.py

```
@@ -33,17 +34,21 @@ def registered_model_rest_api(
+def updated_model_registry_resource(
     request: pytest.FixtureRequest,
     model_registry_rest_url: str,
     model_registry_rest_headers: dict[str, str],
     registered_model_rest_api: dict[str, Any],
 ) -> dict[str, Any]:
-    model_artifact_id = registered_model_rest_api["model_artifact"]["id"]
-    assert model_artifact_id, f"Model artifact id not found: {registered_model_rest_api['model_artifact']}"
+    resource_name = request.param.get("resource_name")
+    api_name = request.param.get("api_name")
+    if not (api_name and resource_name):
+        raise MissingParameter("resource_name and api_name are required parameters for this fixture.")
```

Pylint (3.3.7)

[convention] Line too long (102/100)

(C0301)